pkgs/development/tools/misc/gdb/default.nix

```diff
@@ -70,6 +70,7 @@ stdenv.mkDerivation rec {
     "--with-gmp=${gmp.dev}"
     "--with-mpfr=${mpfr.dev}"
     "--with-expat" "--with-libexpat-prefix=${expat.dev}"
+    "--with-auto-load-safe-path=${stdenv.cc.cc.lib}"
```

Comment:
Or should we perhaps use a `safePaths ? [ stdenv.cc.cc.lib ]` option?

Reply:
I'd prefer this

Reply:
I have not worked much with autoconf; can I just specify multiple `--with-auto-load-safe-path=/some/path` or do I give it a comma/colon-separated list of paths?

Reply:
https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=blob;f=gdb/configure.ac;h=6ba7c5cfa4f1d287255235e2929206df52efe7ef;hb=HEAD#l126

It's apparently `:` and the default value is '$debugdir:$datadir/auto-load'